src/vs/workbench/contrib/accessibility/browser/accessibilityConfiguration.ts

```diff
@@ -14,6 +14,7 @@ export const accessibleViewIsShown = new RawContextKey<boolean>('accessibleViewI
 export const accessibleViewSupportsNavigation = new RawContextKey<boolean>('accessibleViewSupportsNavigation', false, true);
 export const accessibleViewVerbosityEnabled = new RawContextKey<boolean>('accessibleViewVerbosityEnabled', false, true);
 export const accessibleViewGoToSymbolSupported = new RawContextKey<boolean>('accessibleViewGoToSymbolSupported', false, true);
+export const accessibleViewCurrentProviderId = new RawContextKey<string>('accessibleViewCurrentProviderId', undefined, undefined);
```

Comment:
note that this name isn't really fitting because it's actually the verbosity setting key, but I don't think we need to add a provider ID at this time when this can be used instead

Reply:
if there happens to be a secnodary need for an ID, then it'll be worth adding

Reply:
actually, I can trim the setting part off of it .... that's what I'll do